demo-simple/src/main/res/layout/activity_main.xml

```diff
@@ -125,17 +125,9 @@
     android:text="Delete all" />
 
   <Button
-    android:id="@+id/button_show_completed_downloads"
+    android:id="@+id/button_show_files_in_directory"
```

Comment:
Naming doesn't match the text or the variable name in `MainActivity`. If you make these consistent it will be easier to find when looking through the codebase.